src/libostree/libostree-devel.sym

```diff
@@ -23,6 +23,7 @@ global:
     ostree_mutable_tree_fill_empty_from_dirtree;
     ostree_mutable_tree_new_from_checksum;
     ostree_mutable_tree_check_error;
+    ostree_repo_get_min_free_space_bytes;
```

Comment:
Very skeptical about LIBOSTREE_2018 version numbers here. Might cause mismatch up/down stream(until we rebase?). Please advice as necessary

Reply:
Note to myself:
Make sure to match and verify with experts of `debian/libostree-1-1.symbols` in debian-branch:
```
ostree_repo_get_min_free_space_bytes@LIBOSTREE_2018.X 2018.X
```

Reply:
Probably makes sense to create a new section for this symbol, since it’s going to be in the upstream 2018.9 release now that it’s been accepted upstream:
```
LIBOSTREE_2018.9 {
global:
    ostree_repo_get_min_free_space_bytes;
} LIBOSTREE_2018.7;
```

When this patch gets rebased (dropped) when we rebase to the 2018.9 release, the symbol versioning will not change.

That said, I often get symbol versioning wrong, so you’ll have to try my suggestion out and see if it works. The corresponding Debian `libostree-1-1.symbols` line would be something like:
```
ostree_repo_get_min_free_space_bytes@LIBOSTREE_2018.9 2018.6-3endless2
```
inserted in the right place for alphabetical order. The reasoning behind it is that `ostree_repo_get_min_free_space_bytes@LIBOSTREE_2018.9` is the symbol name and version coming from the `.so` file; and `2018.6-3endless2` is the first Debian package version which it appeared in. The idea behind the Debian symbols file is to allow package dependencies to be automatically calculated by looking at symbol dependencies in object files.

Reply:
>That said, I often get symbol versioning wrong, so you’ll have to try my suggestion out and see if it works.

Yes, this is very non-trivial to me. Not even sure how to test that your/mine/x solution even "works" or not.

Reply:
> Not even sure how to test that your/mine/x solution even "works" or not.

Build it!

Reply:
No, I think the `LIBOSTREE_2018.9` version tag needs to depend on the `LIBOSTREE_2018.7` tag (which is present in `libostree-devel.sym`), since that tag has been used to control the symbol exports on our most recent OSTree release. The LD version tags have no semantics, and there’s no formal relationship between them and the project version number. 

To put it another way, making `LIBOSTREE_2018.9` depend on `LIBOSTREE_2018.6` would introduce a branch in the LD dependencies which might cause problems with the symbol versioning stored in modules which link against libostree (like flatpak), where they would depend on one branch but not the other.

Reply:
I see.
It seems that there are some concrete concepts here and I don't really understand them.